src/sbt-test/interplay/library-with-root/test

```diff
@@ -13,6 +13,7 @@ $ exec git remote add origin $PWD/target/remote
 $ exec git push origin main
 $ exec git branch -u origin/main
 
+> reload
```

Comment:
These `reload`s are now necessary because starting with sbt 1.4.0 it seems the initial state of the sbt project does not "see" the above created `.git` folder (created via `git init`) yet. Otherwise following error will occur:
```
[error] java.lang.RuntimeException: Aborting release. Working directory is not a repository of a recognized VCS.
```
which gets thrown [here](https://github.com/sbt/sbt-release/blob/f29b88137dd1db4f99ef5f279a5fe8f8f2b9590c/src/main/scala/ReleaseExtra.scala#L112).
sbt-release [recognizes](https://github.com/sbt/sbt-release/blob/ed1202c399e2447586628aba64775844dc62334f/src/main/scala/Vcs.scala#L53) a VCS by simply looking if a [folder like ".git"](https://github.com/sbt/sbt-release/blob/ed1202c399e2447586628aba64775844dc62334f/src/main/scala/Vcs.scala#L122) does [exist](https://github.com/sbt/sbt-release/blob/ed1202c399e2447586628aba64775844dc62334f/src/main/scala/Vcs.scala#L69-L70).

sbt-release does this with [a "startStage"](https://github.com/sbt/sbt-release/blob/f29b88137dd1db4f99ef5f279a5fe8f8f2b9590c/src/main/scala/ReleasePlugin.scala#L199). Now my guess was that maybe with sbt 1.4.0 that state does not contain the .git folder yet in scripted tests. I am pretty sure my guess is correct, because when I put the reload _before_ the `exec git init` the tests still fail, but if I put it directly after the git init, they succeed.

Reply:
Your guess sounds correct to me as well. Can we have this explanation as a comment in the `test` files, though?

Reply:
Added a comment to each reload.